### ISSUE TITLE
Add webpack loader

### ISSUE THREAD

**Issue by mkusher** [TheBrainFamily/cypress-cucumber-preprocessor#114]
Webpack can be supported by having a loader for `.feature` files, similar to what is already done for browserify.

Current workaround when using webpack is to have both webpack for `.ts` and browserify for `.feature`. Example (https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/issues/18#issuecomment-384245388)
```js
on('file:preprocessor', (file) => {
    if (file.filePath.match(/\.(js|jsx)/g)) {
      return webpack(options)(file)
    } else {
      return cucumber()(file)
    }
})
```

Having loader might be a solution for: https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/issues/112

**Comment by lgandecki** [comment]
I don't understand what you mean, what's the usecase? can you do this as a showcase? I don't want to add things to the codebase just because we can. 
The #112 I believe is invalid, as I (and most people) used typescript with import/export.

**Comment by mkusher** [comment]
it's useful for projects where webpack is already configured for transpiling cypress. Right now cucumber preprocessor will use browserify for `.feature` and everything it will require/import, e.g. step definitions. This means that step definitions won't respect my webpack configuration(and shouldn't since they are transpiled using browserify), so one has to duplicate all configurations for both webpack and browserify

**Comment by mkusher** [comment]
@lgandecki no, I don't share code between tests and app, I just have a configuration for ts paths and webpack aliases that I would like to reuse in both specs and steps

**Comment by lgandecki** [comment]
Closed in #115 , thank you so much !
